src/lib/UI/Config/Provider/FieldType/RichText/AlloyEditor.php

```diff
@@ -57,4 +79,42 @@ protected function getExtraButtons(): array
 
         return $this->alloyEditorConfiguration['extra_buttons'] ?? [];
     }
+
+    /**
+     * Get custom CSS classes defined by the SiteAccess-aware configuration.
+     *
+     * @return array
+     */
+    private function getCssClasses(): array
+    {
+        return $this->onlineEditorConfigMapper->mapCssClassesConfiguration(
+            $this->getSiteAccessConfigArray(RichText::CLASSES_SA_SETTINGS_ID)
+        );
+    }
+
+    /**
+     * Get custom data attributes defined by the SiteAccess-aware configuration.
+     *
+     * @return array
+     */
+    private function getDataAttributes(): array
+    {
+        return $this->onlineEditorConfigMapper->mapDataAttributesConfiguration(
+            $this->getSiteAccessConfigArray(RichText::ATTRIBUTES_SA_SETTINGS_ID)
+        );
+    }
+
+    /**
+     * Get configuration array from the SiteAccess-aware configuration, checking first for its existence.
+     *
+     * @param string $paramName
+     *
+     * @return array
+     */
+    private function getSiteAccessConfigArray(string $paramName): array
+    {
+        return $this->configResolver->hasParameter($paramName)
```

Comment:
This method seems to be redundant (we should have some defaults for `RichText::CLASSES_SA_SETTINGS_ID` and `RichText::ATTRIBUTES_SA_SETTINGS_ID`), but as you said on Slack: "better to be safe than sorry"

Reply:
IMHO this is the safest way to fetch parameters. Default values are configured in yaml, so it might be maintenance-wise error-prone once we all forget about this code in a couple of months